File: Rudder/Classes/RudderServerConfigSource.m
//
//  RudderServerConfigSource.m
//  RudderSDKCore
//
//  Created by Arnab Pal on 17/10/19.
//  Copyright © 2019 Rudderlabs. All rights reserved.
//

#import "RudderServerConfigSource.h"

@implementation RudderServerConfigSource

- (instancetype)init
{
    self = [super init];
    if (self) {
        self.destinations = [[NSMutableArray alloc] init];
    }
    return self;
}

- (void)addDestination:(RudderServerDestination *)destination {
    [self.destinations addObject:destination];
}

@end

File: RudderSampleProject/RudderSdkCore/RudderServerConfigSource.swift
//
//  RudderServerConfigSource.swift
//  RudderSdkCore
//
//  Created by Arnab Pal on 11/10/19.
//  Copyright © 2019 Arnab Pal. All rights reserved.
//

import Foundation

class RudderServerConfigSource : Decodable {
    var config: String? = nil
    var id: String = ""
    var name: String = ""
    var enabled: Bool = false
    var updatedAt: String = ""
    var destinations: [RudderServerDestination] = []
}
